status.go

```diff
@@ -385,6 +385,12 @@ func (vs *version) setState(state versionState) {
 		vs.state = state
 		if state == versionAvailable {
 			vs.available = time.Now()
+
+			if vs.stats != nil {
+				tags := []string{fmt.Sprintf("sequins_db:%s", vs.db.name)}
+				span := vs.available.Sub(vs.created)
```

Comment:
Actually turns out I called this `duration` instead of `span`, since I think spans imply tracing contexts, but I don't think it really matters.